src/Init/Lean/Parser/Parser.lean

```diff
@@ -1822,12 +1822,7 @@ private def catNameToString : Name → String
 | n                           => n.toString
 
 @[inline] def mkCategoryAntiquotParser (kind : Name) : ParserFn :=
--- allow "anonymous" antiquotations `$x` for the `term` category only
--- TODO: make customizable
--- one good example for a category that should not be anonymous is
--- `index` in `tests/lean/run/bigop.lean`.
-let anonAntiquot := kind == `term;
-(mkAntiquot (catNameToString kind) none anonAntiquot).fn
+(mkAntiquot (catNameToString kind) none).fn
```

Comment:
@leodemoura This is probably the only fundamental change outside of the parenthesizer. I noticed that right now we create big choice nodes for tactic antiquotations, exactly like we used to for term antiquotations: if we write `begin $t end` this is a choice of all possible tactics before this commit. We used the bigop test to argue for the current behavior, but I would argue that the tests below actually become clearer and more consistent with this change.